Kitodo/src/main/webapp/WEB-INF/templates/includes/system/migration.xhtml

```diff
@@ -104,6 +111,16 @@
                 update="systemTabView:migrationForm"/>
         </h:panelGroup>
 
+        <h:panelGroup id="ldapManagerPasswordsMigration" rendered="#{MigrationForm.ldapManagerPasswordsMigrationRendered}">
+            <div/>
```

Comment:
Please add some text or explanation here as an empty content is bad.

![Screenshot_2021-09-22_11-44-43](https://user-images.githubusercontent.com/2129672/134321576-c47f70cb-ed76-4aeb-bc46-bf5413843181.png)